app/src/main/java/woowacourse/omok/WinActivity.kt

```diff
@@ -0,0 +1,56 @@
+package woowacourse.omok
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Button
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import omok.domain.state.Turn
+import woowacourse.omok.database.DBController
+import woowacourse.omok.database.OmokConstract
+import woowacourse.omok.database.OmokDBHelper
+
+class WinActivity : AppCompatActivity() {
+    lateinit var dBController: DBController
```

Comment:
```suggestion
    private lateinit var dBController: DBController
```
클래스 내부에서 사용하는 것이라면, private 접근자를 붙이는 습관을 들이는 것이 좋습니다. 😄